### PR TITLE
Fix lunarvim info nil issue

### PR DESCRIPTION
**Pull request by abzcoding**
# Description

fix nill issues in LunarVim info when there are no active LS or the lang is not in `default_config.lua`

### CODE REVIEW

**Comment by abzcoding** [comment]
image

File: lua/core/info.lua [hunk 1]
```diff
@@ -130,13 +130,26 @@ end
...
+  local missing_linters = {}
+  local missing_formatters = {}
   local num_caps = vim.tbl_count(client_enabled_caps)
   local null_ls_providers = null_ls_handler.get_registered_providers_by_filetype(ft)
-
-  local missing_linters = lvim.lang[ft].linters._failed_requests or {}
-  local missing_formatters = lvim.lang[ft].formatters._failed_requests or {}
+  if client ~= nil then
+    is_client_active = not client.is_stopped()
+    client_enabled_caps = require("lsp").get_ls_capabilities(client.id)
```
Comment (PasiBergman) [inline lua/core/info.lua:144]: Should the `num_caps` be calculated again after getting them?

Comment (abzcoding) [reply lua/core/info.lua:144]: yes they should you are right

**Review by PasiBergman — APPROVED** [review]
Looks fine to me.

**Pull request merged**